### PR TITLE
Ibp 4609 variable details popup

### PR DESCRIPTION
**Pull request by lik-83**

### CODE REVIEW

File: src/main/jhipster/src/main/webapp/app/ontology/variable-details/valid-values.component.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { JhiLanguageService } from 'ng-jhipster';
+import { VariableDetails } from '../../shared/ontology/model/variable-details';
+import { VariableDetailsContext } from './variable-details.context';
+
+enum DataType {
+    NUMERIC = 'Numeric',
+    CATEGORICAL = 'Categorical',
+    CHARACTER = 'Character'
+}
```
Comment (nahuel-soldevilla) [inline src/main/jhipster/src/main/webapp/app/ontology/variable-details/valid-values.component.ts:10]: maybe we can move it to `/shared/ontology`

File: src/main/jhipster/src/main/webapp/app/ontology/variable-details/variable-details.component.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		</ul>
+	</nav>
+	<div class="tab-content p-3">
+		<router-outlet></router-outlet>
+		<jhi-alert></jhi-alert>
+	</div>
+</div>
+<div class="modal-footer">
+	<button type="button" class="btn btn-primary" data-dismiss="modal" (click)="dismiss()">
+		<span jhiTranslate="release-notes.ok-button"></span>
```
Comment (nahuel-soldevilla) [inline src/main/jhipster/src/main/webapp/app/ontology/variable-details/variable-details.component.html:26]: we create `global.ok`, we have some similar there already (_confirm, create, back, etc_)

**Review by nahuel-soldevilla — APPROVED** [review]
awesome

File: src/main/jhipster/src/main/webapp/app/ontology/variable-details/valid-values.component.ts (hunk 1 above)
Comment (clarysabel) [reply src/main/jhipster/src/main/webapp/app/ontology/variable-details/valid-values.component.ts:10]: Dont we have DATE data type? Just thinking that when creating a scale, Date is also available and for the germplasm details - attributes section we are requested to handle date data type
@lik-83

Comment (lik-83) [reply src/main/jhipster/src/main/webapp/app/ontology/variable-details/valid-values.component.ts]: I moved to '/shared/ontology'. Regarding the data types values, I just only added the ones that I have used in the popup.

File: src/main/jhipster/src/main/webapp/app/ontology/variable-details/variable-details.component.html (hunk 2 above)
Comment (lik-83) [reply src/main/jhipster/src/main/webapp/app/ontology/variable-details/variable-details.component.html]: Done

**Pull request merged**